src/components/header/header.tsx

```diff
@@ -1,15 +1,22 @@
 import { ReactNode } from 'react';
 import Logo from '../logo/logo';
+import Breadcrumbs from '../breadcrumbs/breadcrumbs';
 
 type HeaderProps = {
   children?: ReactNode;
+  hasBreadcrumbs?: boolean;
```

Comment:
Вместо булева пропса лучше сделать необязательный пропс слот и прокидывать крошки 
```suggestion
  breadcrumbs: ReactNode,
```